src/commands/list.rs

```diff
@@ -4,15 +4,15 @@ use prettytable::{format, Table};
 use crate::config;
 
 pub fn list(shortcut: Option<String>) -> Result<i32> {
-    if let Some(call) = shortcut {
+    if let Some(shortcut) = shortcut {
         let config: config::Config = config::Config::load()?;
 
         let mut shortcut_list = Table::new();
         shortcut_list.set_format(*format::consts::FORMAT_NO_BORDER_LINE_SEPARATOR);
         shortcut_list.set_titles(row!["Shortcuts", "Shortcut Name", "Shortcut Location"]);
```

Comment:
```suggestion
        shortcut_list.set_titles(row!["Name", "Shortcuts", "Shortcut Location"]);
```
I wonder if we should make the Name column first now to signify it's importance.